### ISSUE TITLE
Footer template part not using <footer> tag

### ISSUE THREAD

**Issue by bradhogan** [woocommerce/woocommerce-gutenberg-products-block#6197]
## Describe the bug

My footer template part that on all other (non-woo) pages uses `<footer class="wp-block-template-part">` is using just` <div class="wp-block-template-part">` on Woo pages.

## To reproduce

1. Go to normal page and view source for the footer 
2. Go to a Woo page and view the source 

## Expected behavior

Footer should be wrapped in `<footer> `not in `<div>`

## Screenshots
Normal page (notice footer tag being used):
<img width="396" alt="normal page" src="https://user-images.githubusercontent.com/1781226/161836399-2dcc1d51-62bc-46d5-bffa-f91565b9ba78.png">

Woo page (notice div used instead of footer):
<img width="511" alt="woo page" src="https://user-images.githubusercontent.com/1781226/161836409-b9b6f662-8e25-498b-940b-0e4cb74f8c79.png">

### WordPress (please complete the following information):

-   Core version: 5.9.2
-   Gutenberg version (if installed): 12.9.0
-   WooCommerce version: 6.3.1
-   WooCommerce Blocks version: 7.3.0
-   Site language: English
-   Any other plugins installed: no

### Desktop (please complete the following information):

-   macOS
-   Chrome

**Comment by Aljullu** [comment]
Thanks for raising this issue @bradley2083! I can reproduce it with Twenty Twenty Two.

I guess we should make these changes to all [our templates](https://github.com/woocommerce/woocommerce-blocks/tree/trunk/templates/templates):

```diff
- <!-- wp:template-part {"slug":"header"} /-->
+ <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
...
- <!-- wp:template-part {"slug":"footer"} /-->
+ <!-- wp:template-part {"slug":"footer","tagName":"footer"} /-->
```

I'm not sure if there was a reason why we didn't do that in the first place, but everything seems to work fine for me when I make the change.
